newIDE/app/src/CompactPropertiesEditor/PropertiesMapToCompactSchema.js

```diff
@@ -234,6 +234,37 @@ const createField = (
       getDescription,
       hasImpactOnAllOtherFields: property.hasImpactOnOtherProperties(),
     };
+  } else if (valueType === 'animationname') {
+    function getChoices() {
```

Comment:
The rest of the code uses this notation: `getChoices: () => {`
I think it's a bit easier to read.